### DISCUSSION TITLE
Use displays in Calendar Layout preview

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [directus/directus#7266] [Feature Requests]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

In an attempt to display the status in a formatted form on the calendar, to not display the string, but display only the colored dot (as in the table layout).

This bug is only in the calendar layout, which does not display formatted status, insists on displaying text only.

https://discord.com/channels/725371605378924594/741319150030553159/873228681613561886

### To Reproduce

1. Change layout for calendar
2. Choose status field
3. Check if is showing the string or the green circle

![Captura_de_Tela_2021-08-06_as_17 33 20](https://user-images.githubusercontent.com/379850/128598992-11fa2d61-552d-4745-aa3a-d99c41ea14e8.png)
![Captura_de_Tela_2021-08-06_as_17 33 33](https://user-images.githubusercontent.com/379850/128598993-83d2c25e-4128-4390-ad06-f1dc91afb2dc.png)
![Captura_de_Tela_2021-08-06_as_17 31 46](https://user-images.githubusercontent.com/379850/128598995-b452c1e0-48d7-4ee2-ae0e-da8c8fc9eba6.png)

### What version of Directus are you using?

v9.0.0-rc.88

### What version of Node.js are you using?

node --version v16.5.0

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### What operating system are you using?

macOS, Windows

### How are you deploying Directus?

Docker

**Comment by rijkvanzanten** [comment]
> This bug is only in the calendar layout, which does not display formatted status, insists on displaying text only.

Yup! We pass on the item to FullCalendar (https://fullcalendar.io) to be rendered, which relies on strings for the preview..

We'll treat this as a feature request to support displays on the calendar layout 👍🏻

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
